src/app/loadout/LoadoutDrawer.tsx

```diff
@@ -47,55 +48,47 @@ interface State {
   show: boolean;
   showClass: boolean;
   isNew: boolean;
+  clashingLoadout: Loadout | null;
 }
 
-const typesSelector = createSelector(
```

Comment:
I can't get Prettier to leave this code alone for the life of me. Even after reverting the changes manually and saving the format on commit got ahold of it. I will leave it for now but let me know if you want me to clean it up and I will see what I can work out.